compose/backend.yml

```diff
@@ -2,6 +2,35 @@
 version: '3'
 
 services:
+
+  annotation-api:
+    image: hmctspublic.azurecr.io/em/anno:latest
+    environment:
+      - SPRING_DATASOURCE_URL=jdbc:postgresql://annotation-api-db:5432/emannotationapp
+      - SPRING_DATASOURCE_USERNAME=emannotationapp
+      - SPRING_DATASOURCE_PASSWORD=emannotationapp
+      - S2S_BASE_URI=http://service-auth-provider-api:8080
+      - IDAM_API_BASE_URI=http://idam-api:8080
```

Comment:
This must be sidam-api, right? We are using sidam-api as host name for idam after sidam upgrade.
Well - this line seems to be duplicate already - pls remove.

Reply:
Done.